packages/material-ui/src/Popover/Popover.test.js

```diff
@@ -861,4 +876,166 @@ describe('<Popover />', () => {
       expect(transitionHandler.callCount).to.equal(1);
     });
   });
+
+  describe('prop: onEnter', () => {
+    beforeEach(() => {
+      PropTypes.resetWarningCache();
+      stub(console, 'error');
+    });
+
+    afterEach(() => {
+      console.error.restore();
+    });
+
+    it('issues a warning', () => {
+      PropTypes.checkPropTypes(
+        Popover.Naked.propTypes,
+        {
+          onEnter: () => [],
+        },
+        'props',
+        'Popover',
+      );
+
+      // expect(console.error.callCount).to.equal(1);
```

Comment:
I guess there are other prop types warnings beside the deprecation warning? The Popover might have required props.

Reply:
> The Popover might have required props.

That makes sense, thanks.

> Saw this in another PR of [yours] as well

Yeah, copy-pasta. Fixed, thanks.